bootstrap/009_register.sql

```diff
@@ -0,0 +1,29 @@
+
+create table if not exists internal.external_id_table as (select randstr(16, random()) as external_id);
```

Comment:
Why make a new table vs. adding a row to `internal.config`?

Reply:
To keep it somewhat secret (eg not in the catalog view) and to ensrue it can never be overwritten w/o extra logic

Reply:
I _alwasy_ forget about the config catalog view. Makes sense.